### PR TITLE
Remove Ricochet as it is already discontinued in 2016.

### PR DESCRIPTION
**Pull request by ohmynameisrico**
## The Tor Project said Ricochet is a "Discontinued software" thus it shall be removed.

Software listed here has become depreciated and development has ceased. They are therefore considered unsafe and it is highly recommended to avoid using them. This section is here purely for historical value.
Ricochet IM is an open-source, decentralised instant messenger project that officially ended development in November 2016. Ricochet starts a Tor onion service on the users local system and facilitates communication with other Ricochet users whom are also running their own Ricochet-created Tor onion service, providing End-to-End encryption by never allowing the connection to leave the Tor network.

### HTML Preview

*Replace [GITHUB_USERNAME] with your GitHub username and [BRANCH] with the branch name.*

http://htmlpreview.github.io/?https://github.com/ohmynameisrico/privacytools.io/blob/patch-1/index.html

### CODE REVIEW

**Comment by ohmynameisrico** [comment]
https://github.com/ricochet-im/ricochet/issues/579
https://github.com/privacytoolsIO/privacytools.io/issues/474

**Comment by hugoncosta** [comment]
With the availability of tens of services like this, I guess removing one that doesn't have proper maintenance is a must. @Shifterovich?

**Comment by pahakalle** [comment]
what is going to take its place?

**Comment by beardog108** [comment]
Its not actually [fully] discontinued. I don't think it should be removed unless someone can show these supposed "10 known vulnerabilities", the Tor wiki page doesn't even mark it as discontinued anymore.

Ricochet is relatively simple, and if it had *known* unpatched vulnerabilities, the authors would make it clear that its not safe to use. 

In addition Ricochet (the protocol anyway) is being extended to support group messaging via the [Cwtch project](https://cwtch.im/), so development around it is not completely stopped.

Quote From dev on June 4 2018:

>I reached out to the author of the Ricochet section on that Tor wiki page and cleared things up. You're free to make judgments about development being stalled, and Ricochet's disclaimers are still in place (experimental, etc), but the "officially ended development" claim was incorrect and is factually false.

IMO Ricochet is just as safe/unsafe as the day of its last release, unless someone can link the vulnerabilities that are claimed to be known. Is it the safest messenger available? No, but it is pretty good, and remember, it did pass the security audit in 2016, which was for the then (and still latest) release.

I wouldn't be upset about moving it to "worth mentioning", since it is clearly not well supported. Tox is alright, but has its own sets of problems and is not as private as Ricochet, although since it is supported, it would be better for the featured spot.

# Tl;dr

I suggest moving it to worth mention like suggested, and move Tox to the recommended. 

Hopefully one day Ricochet is better supported, as it is simple yet secure (in concept) and anonymous, which is rare.

**Comment by tya99** [comment]
As mentioned here https://github.com/privacytoolsIO/privacytools.io/issues/474#issuecomment-438347467 I don't think this should be removed.

I also don't think that "Tox" should be argued as a "replacement" they aren't really the same thing and fundamentally work quite differently.

See:

https://github.com/ricochet-im/ricochet/pull/580#issuecomment-394176462
https://github.com/ricochet-im/ricochet/issues/579#issuecomment-392788611

**Comment by asddsaz** [comment]
Without any strong evidence of major security flaws, why should it be deleted?

If you want security with decent ease of use, you use Ricochet. I am unware of any alternative to take its place.

I2p-bote and Bitmessage are the closest other "options".

**Comment by tya99** [comment]
This should be closed along with the duplicate https://github.com/privacytoolsIO/privacytools.io/issues/474
